Create installation instructions for the Intel Edison

gunthercox opened the pull request:
- [ ] Linux setup instructions
- [ ] Windows setup instructions
- [ ] Mac setup instructions

gunthercox committed:
Started writing linux setup instructions.

kevin-brown commented:
Should we set up Jekyll while we're at it? I noticed we've got a lot of shared template space.

gunthercox commented:
It would be a good idea to have jekyll set up. It will prevent a lot of work in the future if we have to go through and change resources or something in the navbar.

kevin-brown committed:
Set up Jekyll and fix small errors in setup.html

kevin-brown committed:
Add section for getting the device

This also switches it so we now use the `<mark>` element for
highlighting parts of the commands that are important and have
additional context that may be useful.

kevin-brown commented:
We now have very basic setup instructions between here and the README, we can continue on this once we figure out how we are going to handle different OSes and devices.
